### ISSUE TITLE
AC Node: add a command to query fan state like we have in the Light Node

### ISSUE THREAD

**Issue by khaledhassan** [khaledhassan/HAS#18]

**Comment by dangyuluo** [comment]
Is it necessary? The fan status is completely controlled by the controller. They should be the same as long as the system is running.

**Comment by khaledhassan** [comment]
What should happen if the controller restarts? Should we do anything special to account for manual control of the actuators via the web interface? I added that feature in the light node so that the controller doesn't think the light is off when it is actually on in case of a controller crash/restart

**Comment by dangyuluo** [comment]
Yeah, that makes sense. We can actively report the fan status in the upload json string.
